testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcOidcBrokerConfigurationUserInfoOnlyMappers.java

```diff
@@ -0,0 +1,82 @@
+package org.keycloak.testsuite.broker;
+
+import org.keycloak.models.IdentityProviderModel;
+import org.keycloak.models.IdentityProviderSyncMode;
+import org.keycloak.protocol.ProtocolMapperUtils;
+import org.keycloak.protocol.oidc.OIDCLoginProtocol;
+import org.keycloak.protocol.oidc.mappers.OIDCAttributeMapperHelper;
+import org.keycloak.protocol.oidc.mappers.UserAttributeMapper;
+import org.keycloak.provider.ProviderConfigProperty;
+import org.keycloak.representations.idm.ClientRepresentation;
+import org.keycloak.representations.idm.ProtocolMapperRepresentation;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.keycloak.testsuite.broker.BrokerTestConstants.*;
+import static org.keycloak.testsuite.broker.BrokerTestTools.*;
+
+/**
+ * @author hmlnarik
+ */
+public class KcOidcBrokerConfigurationUserInfoOnlyMappers extends KcOidcBrokerConfiguration {
+
+    public static final KcOidcBrokerConfigurationUserInfoOnlyMappers INSTANCE = new KcOidcBrokerConfigurationUserInfoOnlyMappers();
+
+    protected static final String ATTRIBUTE_TO_MAP_USER_INFO = "user-attribute-ufo";
+
+
+    @Override
+    public List<ClientRepresentation> createProviderClients() {
+        ClientRepresentation client = new ClientRepresentation();
```

Comment:
Is it possible to call
```
List<ClientRepresentation> clients = super.createProviderClients();
```
at the beginning and then just update the configuration of the particular client based on what is needed for your test? This will allow to remove duplicated code copy/pasted from the parent class. See some other classes, which do the same approach - for example KcOidcBrokerClientSecretJwtTest.KcOidcBrokerConfigurationWithJWTAuthentication.

I suggest to also take the approach with the anonymous configuration class as used by KcOidcBrokerClientSecretJwtTest is also good as the "kcOidcBrokerConfigurationUserInfoOnlyMappers" is used just by your test class and hence it is not needed to introduce another public class "kcOidcBrokerConfigurationUserInfoOnlyMappers" to Keycloak codebase for this.